homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
The problem here is that light/isy994 inherits from both `Light` and `ISYDevice`. Both of those have `state_attributes` property override.

I don't know which implementation is more correct, so I propose to merge the current fix that restores light.isy994 UI state to pre 0.36 and then think of a proper fix.